torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -1003,6 +1003,31 @@ Operation listAppend(const Node* node) {
   };
 }
 
+template <typename TList>
+Operation listPop(const Node* node) {
```

Comment:
This doesn't really need the `node` and you can just use the `std::function<int(Stack&)>` directly (i.e. this turns into 
```cpp
int listPop(Stack& stack) {
  // stuff
  return 0;
}
```